dll/win32/uxtheme/uxtheme.rc

```diff
@@ -34,6 +34,9 @@
 #ifdef LANGUAGE_SK_SK
     #include "lang/sk-SK.rc"
 #endif
+#ifdef LANGUAGE_TR_TR
+    #include "lang/tr-TR.rc"
+#endif
```

Comment:
Added missing lines for you.

Reply:
I didn't know, but now I know, thank you so much